hnix-store-remote/hnix-store-remote.cabal

```diff
@@ -15,26 +15,63 @@ cabal-version:       >=1.10
 
 library
   exposed-modules:     System.Nix.Store.Remote
+                     , System.Nix.Store.Remote.Binary
                      , System.Nix.Store.Remote.Logger
                      , System.Nix.Store.Remote.Protocol
                      , System.Nix.Store.Remote.Types
                      , System.Nix.Store.Remote.Util
 
   build-depends:       base >=4.10 && <5
-                     , base64-bytestring
+                     , cryptonite
```

Comment:
Is it possible to replace `cryptonite` functions with analogous ones from the `cryptohash-*` family? We previously depended on `cryptonite`, but removed it to reduce our dependencies.

Reply:
Looks like it's not needed at all for `-remote` as everything needed is exported from `-core`. Removed